.vscode/tasks.json

```diff
@@ -110,4 +110,4 @@
             "type": "shell"
         }
     ]
-}
```

Comment:
Let's remove such things from commits. Otherwise, it is hard to navigate any significant changes in file history.